### PR TITLE
Release 1.9.26

### PR DESCRIPTION
**Pull request by loosebazooka**
@salguarnieri @patflynn

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+### Version 1.9.26 (September 24th, 2015)
+* Skipped 1.9.25
```
Comment (salguarnieri) [inline RELEASE_NOTES.md]: I'm ok with this being here, but I sort of feel like there might be a better way to say that a version was skipped.

Comment (loosebazooka) [inline RELEASE_NOTES.md]: Updated to match a previous entry like this. But it's not really any different. You got suggestions for this?

Comment (salguarnieri) [inline RELEASE_NOTES.md]: Not really. Saying something *isn't* there is always a bit cumbersome.

**Comment by salguarnieri** [comment]
lgtm

**Pull request merged**